phpmyadmin/content.md

```diff
@@ -0,0 +1,133 @@
+# What is phpMyAdmin?
+
+phpMyAdmin is a free software tool written in PHP, intended to handle the administration of MySQL over the Web. phpMyAdmin supports a wide range of operations on MySQL and MariaDB. Frequently used operations (managing databases, tables, columns, relations, indexes, users, permissions, etc) can be performed via the user interface, while you still have the ability to directly execute any SQL statement.
+
+Run phpMyAdmin with Alpine, Apache and PHP FPM.
+
+%%LOGO%%
+
+# How to use this image
+
+All following examples will bring you phpMyAdmin on `http://localhost:8080` where you can enjoy your happy MySQL administration.
+
+## Credentials
+
+phpMyAdmin does use MySQL server credential, please check the corresponding server image for information how it is setup.
+
+The official MySQL and MariaDB use following environment variables to define these:
+
+-	`MYSQL_ROOT_PASSWORD` - This variable is mandatory and specifies the password that will be set for the `root` superuser account.
+-	`MYSQL_USER`, `MYSQL_PASSWORD` - These variables are optional, used in conjunction to create a new user and to set that user's password.
+
+## Supported Docker hub tags
+
+The following tags are available:
+
+-	`latest`, `fpm`, and `fpm-alpine` are always the most recent released version
```

Comment:
https://github.com/docker-library/official-images/pull/7288 doesn't have any of these tags -- is that an oversight here, or there? :smile:

Reply:
I fixed it ;)